app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/organisation/GrsController.scala

```diff
@@ -233,61 +222,36 @@ class GrsController @Inject() (
   ): Future[Either[ServiceError, Registration]] =
     request.registration.organisationDetails.partnershipDetails match {
       case Some(partnershipDetails) =>
-        partnershipDetails.partnershipType match {
-          case GENERAL_PARTNERSHIP =>
-            for {
-              generalPartnershipDetails <- generalPartnershipGrsConnector.getDetails(journeyId)
-              result <- update { registration =>
-                val updatedOrgDetails = mergePartnershipDetails(organisationDetails =
-                                                                  registration.organisationDetails,
-                                                                generalPartnershipDetails =
-                                                                  Some(generalPartnershipDetails),
-                                                                scottishPartnershipDetails = None,
-                                                                GENERAL_PARTNERSHIP
-                )
-                registration.copy(incorpJourneyId = Some(journeyId),
-                                  organisationDetails = updatedOrgDetails
-                )
-              }
-            } yield result
-          case SCOTTISH_PARTNERSHIP =>
-            for {
-              scottishPartnershipDetails <- scottishPartnershipGrsConnector.getDetails(journeyId)
-              result <- update { registration =>
-                val updatedOrgDetails = mergePartnershipDetails(organisationDetails =
-                                                                  registration.organisationDetails,
-                                                                generalPartnershipDetails = None,
-                                                                scottishPartnershipDetails =
-                                                                  Some(scottishPartnershipDetails),
-                                                                SCOTTISH_PARTNERSHIP
-                )
-                registration.copy(incorpJourneyId = Some(journeyId),
-                                  organisationDetails = updatedOrgDetails
-                )
-              }
-            } yield result
-          case _ => throw new IllegalStateException("Unsupported partnership type")
-        }
+        for {
+          incorporatedPartnershipDetails <- partnershipGrsConnector.getDetails(journeyId)
```

Comment:
Yeah I thought about it because Rob had mentioned just LLP, Limited and Scottish Limited as incorporate but the other two are not.. I was not sure what I should name it, so left it as Incorporated because when we send it to EIS it would be should groupPartnershipDetails... so I thought it would not matter. 
We can drop incorporated but there is already PartnershipDetails

Reply:
I think its worth making the change as it's difficult, especially for newcomers, to get your head around these models. How about in general we essentially swap `incorporated` for `business`. Perhaps go with `partnershipBusinessDetails ` here?

Reply:
That's sounds good, will do that

Reply:
Done